python/ray/rllib/agents/sac/sac.py

```diff
@@ -0,0 +1,213 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import time
+
+from ray.rllib import optimizers
+from ray.rllib.agents.agent import Agent, with_common_config
+from ray.rllib.agents.sac.sac_policy_graph import SACPolicyGraph
+from ray.rllib.utils.annotations import override
+
+
+OPTIMIZER_SHARED_CONFIGS = [
+    # "buffer_size", "prioritized_replay", "prioritized_replay_alpha",
+    # "prioritized_replay_beta", "prioritized_replay_eps", "sample_batch_size",
+    # "train_batch_size", "learning_starts"
+]
+
+
+# yapf: disable
+# __sphinx_doc_begin__
+DEFAULT_CONFIG = with_common_config({
+    # === Model ===
+    "policy": {
```

Comment:
Other models have flat configs. Is it ok to keep these nested or should I flatten them?

Reply:
Nesting is fine for new configs since we do deep updates, but for common confs like "num_workers" they should be in the same depth (flat). Otherwise, there will be two "num_workers" configs, one of which has no effect, which will be confusing.